src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java

```diff
@@ -98,4 +100,23 @@ public void shouldNotLogLongLinks() throws Exception {
         verify(checkConfig, getPath("InputLineLengthLongLink.java"), expected);
     }
 
+    @Test
+    public void countUnicodePointsOnce() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(LineLengthCheck.class);
+        checkConfig.addAttribute("max", "100");
+        // we need to set charset to let test pass when default charset is not UTF-8
+        final DefaultConfiguration checkerConfig = createRootConfig(null);
+        checkerConfig.addAttribute("charset", StandardCharsets.UTF_8.name());
+        checkerConfig.addChild(checkConfig);
```

Comment:
Why not `createRootConfig(checkConfig)`?

Reply:
I wrote this to emulate the structure of the *checks.xml files.  Would you like me to change it?

Reply:
> createRootConfig(null);

The reason root config has the parameter is to receive the child that will be connected to it. If we can remove the call to `addChild` and just pass the child to `createRootConfig` then we should. It all should amount to the same thing in the end.

Reply:
Oh, yes, of course.  That makes sense. I will update and push right away, thanks for explaining that.